<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>锂电池充放电控制系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton1260">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>输出12.6（12伏锂电池）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1500">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>输出15（12铅酸蓄电池）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeviceInfo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询设备IDN信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_displayError">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询出错信息</string>
    </property>
   </widget>
   <widget class="QListWidget" name="recvmsg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>621</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>收到回复</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeviceInfo_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>启用远程修改控制</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Vset">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_500">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>输出5（3.7伏锂电池）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>输出电压</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setV">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>输出电流</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Cset">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeviceInfo_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>切回本地控制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MeaCurr">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>测量电流</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MeaPow">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>测量功率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MeaVol">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>测量电压</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MeaCap">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>充放电的容量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>返回测量的电压,电流和功率，安时，瓦时</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_outpon">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开输出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_outpoff">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭输出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pri_Vol">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>电压优先</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pri_Cur">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>电流优先</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OutpDelay">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>定时输出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>输出时长</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Cset_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>秒</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>电池充电测试功能</string>
    </property>
    <widget class="QPushButton" name="pushButton_shutPWM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭PWM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trigerKey">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置按键触发</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>充电电压</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chargeV">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>15.00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_setChargeV">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_setChargeC">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chargeC">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>充电电流</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>充电时间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_setChargeT">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chargeT">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_shutCap">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_shutCap">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_shutV">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_shutV">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>16.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>关断电流</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>关断容量</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_shutC">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_shutC">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.05</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>关断电压</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_battOn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开启充电模式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_busTrig">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>充电BUS触发</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trigerBUS">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置BUS触发</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_clear_reply">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除显示</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipEdit">
       <property name="text">
        <string>192.168.123.4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>请输入ip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portEdit">
       <property name="text">
        <string>30000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>请输入端口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop">
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="menu_open"/>
    <addaction name="menu_close"/>
   </widget>
   <widget class="QMenu" name="menu_config">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="menu_modify"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_config"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="menu_modify"/>
  </widget>
  <action name="menu_close">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="menu_modify">
   <property name="text">
    <string>修改</string>
   </property>
  </action>
  <action name="menu_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
